src/pkg/services/m365/m365_test.go

```diff
@@ -48,3 +48,26 @@ func (suite *M365IntegrationSuite) TestUsers() {
 		})
 	}
 }
+
+func (suite *M365IntegrationSuite) TestSites() {
+	ctx, flush := tester.NewContext()
+	defer flush()
+
+	var (
+		t    = suite.T()
+		acct = tester.NewM365Account(suite.T())
+	)
+
+	sites, err := Sites(ctx, acct, fault.New(true))
+	require.NoError(t, err)
+	require.NotNil(t, sites)
+	require.Greater(t, len(sites), 0)
```

Comment:
Nits: 
* `require.` should only be used if the test cannot run beyond this failure.  This should be an assert.
* the order of values should be `assert.Foo(t, expected, result)`.  So in this case: `assert.Less(t, 0, len(sites)).
* Greater/Less is indirect; you probably want `NotEmpty` instead.
```suggestion
	assert.NotEmpty(t, sites)
```

Reply:
Done. Thank you!